dataeng/resources/answer-distribution.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY * 2 ))
+fi
+NUM_REDUCE_TASKS=$(( $NUM_REDUCE_TASKS * 2 ))
+
+now=$(date +%H)
+
+name=hourly
+src="$SOURCES"
+dest=${DESTINATION_PREFIX}/$now
+if [ -z "$MANIFEST_PREFIX" ]; then
+    MANIFEST_PREFIX="$DESTINATION_PREFIX"
+fi
+manifest=${MANIFEST_PREFIX}/$now/manifest.txt
+
+# Remove any previous intermediate output for this hour
+pip install awscli
+aws s3 rm --recursive $dest || true
+aws s3 rm $manifest || true
+
+${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
+  AnswerDistributionWorkflow --local-scheduler \
+  --src $src --dest $dest --name $name --output-root $OUTPUT_URL \
+  --include '[\"*tracking.log*20131*.gz\",\"*tracking.log*2014*.gz\",\"*tracking.log*2015*.gz\",\"*tracking.log*2016*.gz\",\"*tracking.log*2017*.gz\",\"*tracking.log*2018*.gz\"]' \
```

Comment:
We only have to update this file once a year! : )

Reply:
hah, yea, i don't necessarily know what to do about that.  Also the first file looks like it has a typo in it (the original one did too).